src/Store/TaggedStore.php

```diff
@@ -121,14 +127,26 @@ public function fetchOrMakeTagHashes(array $tags, ?int $ttl = null) : array
         $tagHashes = [];
         $ttlTag = '';
         if ($ttl !== null) {
-            $ttlTag = __METHOD__.":ttl:$ttl:".$this->generateHash();
+            $ttlTag = implode(
+                $this->specialKeyDelimeter,
+                [
+                    self::TTL_CACHE_PREFIX,
```

Comment:
This change should probably be considered a breaking change, since it would invalidate all existing TTL tags.

Reply:
True. There are currently 0 production users of this package though so we can get away with it this time if we want to.